app/javascript/mastodon/features/account/components/user_note.js

```diff
@@ -53,33 +50,22 @@ class Header extends ImmutablePureComponent {
     let action_buttons = null;
     if (isEditing) {
       action_buttons = (
-        <div>
-          <IconButton
-            icon='check'
-            title={intl.formatMessage(messages.save)}
-            onClick={this.props.onSaveUserNote}
-            disabled={isSubmitting}
-            size={15}
-          />
-          <IconButton
-            icon='times'
-            title={intl.formatMessage(messages.cancel)}
-            onClick={this.props.onCancelUserNote}
-            disabled={isSubmitting}
-            size={15}
-          />
+        <div className='account__header__user-note__buttons'>
+          <button className='text-btn' tabIndex='0' onClick={this.props.onCancelUserNote} disabled={isSubmitting}>
+            <Icon id='times' size={15} /> <FormattedMessage id='user_note.cancel' defaultMessage='Cancel' />
+          </button>
+          &nbsp;&nbsp;&nbsp;
```

Comment:
Why you preferred `&nbsp;` instead of, say, `margin-left: 5px` on every child of `account__header__user-note__buttons`?

Reply:
laziness and not having to do anything for supporting RTL layouts